test/testXliff20.js

```diff
@@ -596,21 +597,21 @@ module.exports.xliff = {
         var actual = x.serialize();
         var expected =
             '<?xml version="1.0" encoding="utf-8"?>\n' +
-            '<xliff version="2.0" srcLang="en-US" trgLang="de-DE" \n' +
-            '  xmlns:l="http://ilib-js.com/loctool">\n' +
-            '  <file original="foo/bar/asdf.java" l:project="androidapp">\n' +
-            '    <unit id="1" name="foobar" type="res:string" l:datatype="plaintext">\n' +
-            '      <segment>\n' +
-            '        <source>Asdf asdf</source>\n' +
-            '      </segment>\n' +
-            '    </unit>\n' +
-            '  </file>\n' +
+            '<xliff version="2.0" srcLang="en-US" trgLang="de-DE" xmlns:l="http://ilib-js.com/loctool">\n' +
+            '  <file original="foo/bar/asdf.java" l:project="androidapp"/>\n' +
```

Comment:
This is not a correct expected xliff. The first resource string is in project androidapp and in file asdf.java, but that "unit" does not exist inside of this "file" element.

It should be:

```xml
<file original="foo/bar/asdf.java" l:project="androidapp">
  <group id="group_1" name="plaintext">
    <unit id="1" name="foobar" type="res:string" l:datatype="plaintext">
      <segment>
        <source>Asdf asdf</source>
      </segment>
    </unit>
  </group>
</file>
<file original="foo/bar/j.java" l:project="webapp">
  <group id="group_2" name="plaintext">
    <unit id="2" name="huzzah" type="res:string" l:datatype="plaintext">
      <segment>
        <source>baby baby</source>
      </segment>
    </unit>
  </group>
</file>
```